src/Avalonia.Base/Input/DataFormats.cs

```diff
@@ -18,7 +19,10 @@ public static class DataFormats
         /// <summary>
         /// Dataformat for one or more filenames
         /// </summary>
-        [Obsolete("Use DataFormats.Files, this format is supported only on desktop platforms."), EditorBrowsable(EditorBrowsableState.Never)]
+        /// <remarks>
+        /// This data format is supported only on desktop platforms.
+        /// </remarks>
+        [Unstable("Use DataFormats.Files, this format is supported only on desktop platforms. And it will be removed in 12.0"), EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
```suggestion
        [Unstable("Use DataFormats.Files, this format is supported only on desktop platforms. And it will be removed in 12.0."), EditorBrowsable(EditorBrowsableState.Never)]
```